nav/stepper/motion.py

```diff
@@ -18,8 +17,35 @@ def __init__(self, GPIO):
 		[0.0.0] Rishav
 			--- Created function
 	"""
-	def mov(self, dir, distance):
-		print(dir, distance)
+	def mov(self, dir, distance, GPIO):
```

Comment:
Have move take delay as a parameter that defaults to our constant STEPPER_DELAY in case we need to move slower for some reason. 
I don't know if you know about default parameters in python. They look like this: 
def function(default=value):